src/traces/scattermapbox/convert.js

```diff
@@ -331,3 +357,32 @@ function blankFillFunc() { return ''; }
 function isBADNUM(lonlat) {
     return lonlat[0] === BADNUM;
 }
+
+function createClusterColors(colors, steps) {
+    var idx, colors_;
```

Comment:
Non-blocking:
But it would be great if you rename `colors_` temporary variable to `_colors` or even better `newColors`.
Similarly `sizes_` > `newSizes`